presto-hive/pom.xml

```diff
@@ -630,13 +630,15 @@
         </profile>
         <profile>
             <id>test-hive-pushdown-filter-queries-basic</id>
+            <properties>
+                <air.test.jvmsize>5g</air.test.jvmsize>
```

Comment:
We added this because we saw regular failures for this test. Was it only only JDK 17 though ? Or was it also on JDK 8 ?

Reply:
Nevermind, I see we removed L639 `argLine` and replace with this